LayoutTests/fast/scrolling/rtl-bt-drag-scroller.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../resources/js-test-pre.js"></script>
```

Comment:
You can just use `js-test.js` here and remove the `-post.js` file from the bottom.